tests/testthat/test-logo.R

```diff
@@ -1,8 +1,19 @@
-test_that("use_logo() doesn't error", {
+test_that("use_logo() doesn't error with no README", {
   skip_if_not_installed("magick")
   skip_on_os("solaris")
 
   create_local_package()
   img <- magick::image_write(magick::image_read("logo:"), "logo.png")
   expect_no_error(use_logo("logo.png"))
 })
+
+test_that("use_logo() shows a clickable path with README", {
+  skip_if_not_installed("magick")
+  skip_on_os("solaris")
+
+  create_local_package()
+  use_readme_md()
+  img <- magick::image_write(magick::image_read("logo:"), "logo.png")
+  withr::local_options(usethis.quiet = FALSE)
+  expect_snapshot(use_logo("logo.png"),  transform = scrub_testpkg)
```

Comment:
Is there a way to actually test for "clickable path"? Because it doesn't feel perceivable from the current snapshot.

Reply:
Well the single quotes in the snapshot indicates that some markup was applied (and that it could actually find README.md) 'README.md'.